app/src/main/java/org/mozilla/tv/firefox/webrender/VideoVoiceCommandMediaSession.kt

```diff
@@ -317,7 +321,7 @@ class VideoVoiceCommandMediaSession @UiThread constructor(
                 return true
             }
 
-            return super.onMediaButtonEvent(mediaButtonEvent)
+            return super.onMediaButtonEvent(mediaButtonEvent!!)
```

Comment:
Is mediaButtonEvent instance garuntee'd here?

Reply:
**Thank you for catching this!**  I meant to put a TODO on that and look into it later, and completely forgot.  Really bad mistake on my part.